cli.js

```diff
@@ -31,6 +34,13 @@ function run() {
       process.cwd(),
     ], {stdio: 'inherit'});
     break;
+  case 'link':
+    if (!args[1]) {
+      console.error('Usage: react-native link <path>');
```

Comment:
I'm thinking about changing the signature something more like `react-native link MyProject[.xcodeproj] path/to/lib[.xcodeproj]` so it doesn't rely on any magic, and if we decide to something easier, we can build on top of it.